revolv/static/home.scss

```diff
@@ -52,6 +62,46 @@ body {
 
 // --- Cover Image Styles ----------------------------------------------------------------
 
+.cover {
+    width: 100%; 
+    height: 600px;
+    overflow: hidden;
+    position: relative;
+
+    h1 {
+        color: white;
+        text-align: center;
+        text-shadow: 2px 2px black;
+    }
+
+    p {
+        font-size: $revolv-cover-inset-paragraph-size;
+        line-height: $revolv-cover-inset-paragraph-size + 6px;
+        text-align: center;
+        text-shadow: 2px 2px black;
+        font-weight: 200;
+        color: white;
+        padding-top: 30px;
+        line-height: 30px;
+        margin: 0;
+    }
+
+    .shade {
+        @include dark-shade(0.3);
+    }
+
+    .cover-inset {
+        @include absolute-center();
+        width: 600px;
+        z-index: 1;
```

Comment:
Is the z-index: 1 needed? Or was that still from trying to get the svg to show up?